### ISSUE TITLE
There is no option to "Copy Shape" between different instances of Image Editor

### ISSUE THREAD

**Issue by RamyRafik** [ShareX/ShareX#7268]
Currently, with a Shape selected, pressing Ctrl+C will copy its position and dimensions
![image](https://github.com/ShareX/ShareX/assets/20221435/2404a5bc-2dfb-4bdf-90ef-28d337dbe0a1)

Ctrl+C is expected to copy the content of the selected element. Ctrl+V after that should paste it, preferably in the same spot as the source

**Comment by deadlydog** [comment]
I agree. From v16.0.0 [in the changelog](https://getsharex.com/changelog)

> In image editor use Ctrl + C instead of Ctrl + Shift + C to copy image to clipboard

I think it's a normal assumption that `Ctrl` + `C` would copy the selected shape so you can paste another instance of it, rather than copying the entire image. I believe it used to work this way, and it was jarring moving to v16 where this functionality changed.

I would advocate for changing Copy Image back to `Ctrl` + `Shift` + `C` so that `Ctrl` + `C` can be implemented as people would expect it to work. Also, [the keybindings page](https://getsharex.com/docs/keybinds) still lists `Ctrl` + `Shift` + `C` as Copy Image.

I was going to report the new `Ctrl` + `C` keybinding as an issue, but I think it fits well into this issue.
